rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -22,8 +22,9 @@
 
 from __future__ import print_function
 import os
-
+import re
```

Comment:
You are missing empty line here to separate standard library modules from third party ones.